<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>800</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>800</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>800</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="student_tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>780</width>
      <height>780</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>780</width>
      <height>780</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>780</width>
      <height>780</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #808080</string>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Personal Information</string>
     </attribute>
     <widget class="QLabel" name="studentPassword">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>140</y>
        <width>100</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #ffffff</string>
      </property>
      <property name="text">
       <string>Password:</string>
      </property>
     </widget>
     <widget class="QLabel" name="studentNumber">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>180</y>
        <width>100</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #ffffff</string>
      </property>
      <property name="text">
       <string>Student Number:</string>
      </property>
     </widget>
     <widget class="QLabel" name="studentFirstName">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>60</y>
        <width>100</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #ffffff</string>
      </property>
      <property name="text">
       <string>First Name:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="First_name_lineEdit">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>60</y>
        <width>551</width>
        <height>22</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:#ffffff
</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="studentLastName">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>100</y>
        <width>100</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #ffffff</string>
      </property>
      <property name="text">
       <string>Last Name:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="LastName_lineEdit">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>100</y>
        <width>551</width>
        <height>22</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:#ffffff</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_Stu_number">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>180</y>
        <width>551</width>
        <height>22</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #ffffff</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_Password_stu">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>140</y>
        <width>551</width>
        <height>22</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #ffffff</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_explanation_password">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>410</y>
        <width>400</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #ffffff</string>
      </property>
      <property name="text">
       <string>If you want to change your password, you can follow the steps</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_new_pass">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>510</y>
        <width>100</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #ffffff</string>
      </property>
      <property name="text">
       <string>New Password:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_new_pass">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>510</y>
        <width>551</width>
        <height>22</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:#ffffff</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_current_pass">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>470</y>
        <width>551</width>
        <height>22</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:#ffffff
</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_current_pass">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>470</y>
        <width>100</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #ffffff</string>
      </property>
      <property name="text">
       <string>Current Pasword:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_update_pass">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>580</y>
        <width>181</width>
        <height>28</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #ffffff</string>
      </property>
      <property name="text">
       <string>Update</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Lessons</string>
     </attribute>
     <widget class="QLineEdit" name="lineEdit_4">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>300</y>
        <width>520</width>
        <height>150</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:#ffffff
</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>325</x>
        <y>140</y>
        <width>150</width>
        <height>40</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #ffffff</string>
      </property>
      <property name="text">
       <string>Lessons&amp;Teachers</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>Grades</string>
     </attribute>
     <widget class="QPushButton" name="pushButton_3">
      <property name="geometry">
       <rect>
        <x>890</x>
        <y>310</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Remove</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>325</x>
        <y>140</y>
        <width>150</width>
        <height>40</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #ffffff</string>
      </property>
      <property name="text">
       <string>Lessons&amp;Grades</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_5">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>300</y>
        <width>520</width>
        <height>150</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:#ffffff
</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
